frontend/src/components/uam-configure-group/uam-configure-group.tsx

```diff
@@ -111,6 +111,7 @@ const UAMConfigureGroup: FC = () => {
           name={GroupConfigureFieldsName.NAME}
           placeholder="Enter group name"
           type="text"
+          inputClassName={styles.nameField}
```

Comment:
```suggestion
```
styles should be the same as on the `sign-in/up` page. it should be a common styles for the input

Reply:
I've left default styles and updated only background colour as the default input and table backgrounds are the same (gray-300). Or changing default Input background to gray-200 would be better?

Reply:
@what1s1ove but that what ticket was about. It looks like this: 
![Uploading Screenshot 2022-08-30 at 09.22.47.png…]()

Reply:
got it